package.json

```diff
@@ -19,15 +19,12 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.12.x"
-  },
   "dependencies": {
+    "react": "0.12.x",
```

Comment:
Ok, just re-read the `peerDependencies` blog post here: http://blog.nodejs.org/2013/02/07/peer-dependencies/.

I was thinking of it in this context:

> Most plugins never actually depend on their host package, i.e. grunt plugins never do require("grunt"), so even if plugins did put down their host package as a dependency, the downloaded copy would never be used.

But that post specifies that `peerDependencies` should still be used even if your package has a direct dependency on the package in question:

> Even for plugins that do have such direct dependencies, probably due to the host package supplying utility APIs, specifying the dependency in the plugin's package.json would result in a dependency tree with multiple copies of the host package—not what you want.

Will switch back to `peerDependencies`. Good point though, need to set up dist for the non webpack/browserify crowd.